contracts/v2/paranets/ParanetIncentivesPoolFactory.sol

```diff
@@ -54,26 +55,28 @@ contract ParanetIncentivesPoolFactory is Named, Versioned, ContractStatusV2, Ini
     ) external onlyKnowledgeAssetOwner(paranetKAStorageContract, paranetKATokenId) returns (address) {
         HubV2 h = hub;
         ParanetsRegistry pr = paranetsRegistry;
+        string memory incentivesPoolType = isNativeReward ? "Neuroweb" : "NeurowebERC20";
```

Comment:
Can this be more generic, I guess there will be more implementations supported in the future. And should this code work also for other ERC20 tokens?